.github/workflows/ci.yml

```diff
@@ -81,3 +81,29 @@ jobs:
       # Ensure proto generation doesn't depend on external packages.
       - name: Verify proto
         run: make verify-proto
+
+  docker:
+    name: Docker
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Determine tag
+        uses: haya14busa/action-cond@v1
+        id: imagetag
+        with:
+          cond: ${{ github.event_name == 'pull_request' }}
+          if_true: ${{ github.sha }}
+          if_false: "master"
+
+      - name: Build
```

Comment:
A bit of magic what happens here (to me), does this push the server and latest tags as well?

Reply:
The reason for the magic is not to push *latest* for the master branch. (The action does that by default for the master branch)

Reply:
Great 👍